services/scoop/scoop-license.tester.js

```diff
@@ -0,0 +1,93 @@
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('License (valid) - with nested response')
+  .get('/l/ngrok.json')
```

Comment:
you don't need the base route when using `createServiceTester`
fox example here `/l/ngrok.json` can be only `/ngrok.json`

Reply:
Just clicked on resolve and that triggered the build. Haven't made the changes. I'll let you know once I'm done, Johnathan.

Reply:
made changes as part of - https://github.com/badges/shields/pull/10627/commits/e7e40bb0f1af9bdd19cefde1ed43016720ebfcc5